innopoints/schemas/activity.py

```diff
@@ -76,6 +82,7 @@ def get_existing_application(self, activity):
             return appl_schema.dump(application)
         return None
 
+    # pylint: disable=no-member
```

Comment:
Instead of disabling "no-member" in all schemas, just use either use `ignored-classes` (to disable any member checking for Marshmallow objects) or `generated-members` (explicitly listing the members that are dynamically generated) in .pylintrc